package.json

```diff
@@ -310,7 +310,8 @@
   "resolutions": {
     "**/xmlhttprequest-ssl": "~1.6.2",
     "minimist": "^1.2.6",
-    "underscore": "^1.12.1"
+    "underscore": "^1.12.1",
+    "@nguniversal/builders/@angular-devkit/architect": "~0.1501.0"
```

Comment:
isn't that a mistake? @Coly010

Reply:
No, until @nguinversal releases a new version, this is required

Reply:
which they have now, so i'll create a follow up PR